packages/oft-sand/.solcover.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  mocha: {
+    grep: '@skip-on-coverage', // Find everything with this tag
+    invert: true, // Run the grep's inverse set.
+  },
+  skipFiles: ['mock'],
```

Comment:
Since coverage is failing, let's add `sand` here to `skipFiles`, since we want to test the oft-sand contracts specifically, not the sand files which are essentially replicated from `core`

Reply:
done